### PR TITLE
Add blog post: "Parquet: Crafting Data Bridges for Efficient Computation"

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds a blog post titled: "Parquet: Crafting Data Bridges for Efficient Computation" covering the Parquet format, notable features of the format, and how to use it.

Thanks in advance for any thoughts and feedback on this!

### CODE REVIEW

File: _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+
+Parquet files may leverage compression to help reduce file size and increase data read performance.
+Compression is applied at the page level, combining benefits from various encodings.
+Data stored through Parquet is usually compressed when it is written, denoting the compression type through the filename (for example: `filename.snappy.parquet`).
+[Snappy](https://en.wikipedia.org/wiki/Snappy_(compression)) is often used as a common compression algorithm for Parquet data.
+[Brotli](https://en.wikipedia.org/wiki/Brotli), [Gzip](https://en.wikipedia.org/wiki/Gzip), [ZSTD](https://en.wikipedia.org/wiki/Zstd), [LZ4](https://en.wikipedia.org/wiki/LZ4_(compression_algorithm)) are also sometimes used.
+It's worth exploring what compression works best for the data and systems you use (for example, [ZSTD compression may hold benefits](https://www.uber.com/blog/cost-efficiency-big-data)).
+
+
+### <i class="fas fa-icons"></i> "strongly-typed" Data
```
Comment (falquaddoomi) [inline _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md:200]:
```suggestion
### <i class="fas fa-icons"></i> "Strongly-typed" data
```

(Just changed to be consistent with the other headers' capitalization.)

File: _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+# -- schema metadata --
+# data-producer: 'CU DBMI SET Blog'
+```
+
+_Metadata are treated as a distinct and customizable components of Parquet files._
+{:.center}
+
+The Parquet format treats data about the data (metadata) separately from that of column value data.
+[Parquet metadata](https://parquet.apache.org/docs/file-format/metadata/) includes column names, data types, compression, various statistics about the file, and custom fields (in key-value form).
+This metadata may be read without reading column value data which can assist with data exploration tasks (especially if the data are large).
```
Comment (falquaddoomi) [inline _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md:313]: This is a really cool feature!

**Review by falquaddoomi — COMMENTED** [review]
Nice work! I left a few small comments; one of them is a typo, which is why I didn't approve outright, but I don't think the others necessarily need to be addressed. I think this is a truly useful and informative writeup on the format and how to use it. Kudos!

File: _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+# Note: while this is an Arrow in-memory data exception, it also
+# prevents us from attempting to perform incompatible operations
+# within the Parquet file.
+```
+
+_Data value must be all of the same type within a Parquet column._
+{:.center}
+
+Data within Parquet is considered ["strongly-typed"](https://en.wikipedia.org/wiki/Strong_and_weak_typing), entailing specific data types (such as integer, string, etc.) associated with each column and value.
+Attempting to store a data value type which does not match the column data type will usually result in an error (or implied conversion).
```
[the remark this answers is not in the document]
Comment (d33bs) [reply _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md:229]: Thanks - this is very apt! The implied conversion should be a separate topic, as you mention.

File: _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md (hunk 2 above)
Comment (d33bs) [reply _posts/2024-03-25-Parquet-Crafting-Data-Bridges-for-Efficient-Computation.md:313]: Agreed!

**Comment by d33bs** [comment]
Thanks @falquaddoomi for the review! I've made some edits based on what you suggested, including the mention of "parquetry" which felt apt 🙂 .

**Review by falquaddoomi — APPROVED** [review]
I took a look at the recent changes and all's good for me. Love the picture of the parquet flooring and the reference, even if we can't establish that it's the source of the name. Also, the sentiment about constructing a beautiful floor is apt, as the choice of data format is foundational like a floor. Nice work!

**Comment by d33bs** [comment]
Thanks again @falquaddoomi ! Merging this in.

**Pull request merged**